.vscode/launch.json

```diff
@@ -1,403 +1,445 @@
 {
-    // Use IntelliSense to learn about possible attributes.
-    // Hover to view descriptions of existing attributes.
-    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
     "version": "0.2.0",
     "configurations": [
         {
-            "type": "lldb",
+            "type": "codelldb",
             "request": "launch",
-            "name": "Debug unit tests in library 'libp2p-networking'",
             "cargo": {
                 "args": [
                     "test",
                     "--no-run",
-                    "--lib",
-                    "--package=libp2p-networking"
+                    "--package=hotshot-testing",
+                    "--features=full-ci,channel-async-std",
+                    "centralized_server_network"
                 ],
                 "filter": {
-                    "name": "libp2p-networking",
+                    "name": "hotshot-testing",
                     "kind": "lib"
                 }
             },
-            "args": [],
-            "cwd": "${workspaceFolder}"
+            "program": "${cargo:program}",
+            "cwd": "${workspaceFolder}",
+            "name": "Debug centralized_server_network test"
         },
         {
-            "type": "lldb",
+            "type": "codelldb",
             "request": "launch",
-            "name": "Debug example 'clichat'",
             "cargo": {
                 "args": [
-                    "build",
-                    "--example=clichat",
-                    "--package=libp2p-networking"
+                    "test",
+                    "--no-run",
+                    "--package=hotshot-testing",
+                    "--features=full-ci,channel-async-std",
+                    "centralized_server_network_vrf"
                 ],
                 "filter": {
-                    "name": "clichat",
-                    "kind": "example"
+                    "name": "hotshot-testing",
+                    "kind": "lib"
                 }
             },
-            "args": [],
-            "cwd": "${workspaceFolder}"
+            "program": "${cargo:program}",
+            "cwd": "${workspaceFolder}",
+            "name": "Debug centralized_server_network_vrf test"
         },
         {
-            "type": "lldb",
+            "type": "codelldb",
```

Comment:
Only `lldb` instead of `codelldb` works for me.  I'm using the linked debugging extension in the README.md.

Reply:
Changed to lldb!